fastlane/lib/fastlane/actions/oclint.rb

```diff
@@ -204,7 +206,12 @@ def self.available_options
                                        env_name: "FL_OCLINT_ALLOW_DUPLICATED_VIOLATIONS",
                                        description: "Allow duplicated violations in the OCLint report",
                                        is_string: false,
-                                       default_value: false)
+                                       default_value: false),
+          FastlaneCore::ConfigItem.new(key: :extra_arg,
+                                       env_name: 'FL_OCLINT_EXTRA_ARG',
+                                       description: 'Additional argument to append to the compiler command line',
+                                       default_value: '',
```

Comment:
This probably should not have a `default_value` if it is optional, correct?

Reply:
Correct :)